customers.sql

```diff
@@ -0,0 +1,8 @@
+select 
+o.customer_id, c.name, c.email, min(o.created_at) as first_order_at, count(o.id) as number_of_orders
+from `analytics-engineers-club.coffee_shop.customers` c
+join `analytics-engineers-club.coffee_shop.orders` o
+  on c.id = o.customer_id
+group by o.customer_id, c.name, c.email
```

Comment:
I prefer to use `order by 1,2,3` but thats a style thing

Reply:
I'm honestly torn with this... on the one hand it's so much easer.. on the other hand, not all sql dialects support this (cough MSSQL). Also, explicitly calling out column names in the group by makes it easier to interpret a query the first time you read it if you're one of those people who starts reading from the `from` first